pkgs/applications/virtualization/virt-manager/default.nix

```diff
@@ -25,6 +25,8 @@ python3.pkgs.buildPythonApplication rec {
 
   buildInputs = [
     wrapGAppsHook
```

Comment:
Ok. You can came to the same conclusion. `wrapGAppsHook` can be moved to nativeBuildInputs if we are already at it.

Reply:
Yes, have moved the hook as in your PR.